### PR TITLE
Navigate to Unit Overview page from Finish button on codebridge levels

### PR DESCRIPTION
**Pull request by cnbrenci**
This change replaces a no-op in the OnFinish handler for the Finish buttons in codebridge levels with a navigation to the Unit Overview page.

## Links

- jira ticket: [Have finish button direct to unit overview page](https://codedotorg.atlassian.net/browse/CT-664)

## Testing story

Tested locally by adding a pythonlab level to allthethings as the last level and not submittable. This enabled the Finish button for that level. 

I am actually not going to check in that new level with this change since it caused some other unintentional changes in the allthethings level that I'm not sure about, and I don't want to cause merge conflicts with other allthethings changes.  I'm not sure if it's safer to do these types of changes from levelbuilder? Open to feedback on this for sure.

## Follow-up work

Potentially add a test level in the allthethings script for pythonlab.

### CODE REVIEW

File: apps/src/codebridge/InfoPanel/ValidatedInstructions.tsx [hunk 1]
```diff
@@ -99,15 +100,17 @@ const ValidatedInstructions: React.FunctionComponent<InstructionsProps> = ({
...
   const dispatch = useAppDispatch();
 
   const {theme} = useContext(ThemeContext);
 
   const vertical = layout === 'vertical';
 
   const onFinish = () => {
-    // no op for now
-    // Tracked here: https://codedotorg.atlassian.net/browse/CT-664
+    navigateToHref(linkWithQueryParams(`/s/${scriptName}`));
```
Comment (molly-moen) [inline apps/src/codebridge/InfoPanel/ValidatedInstructions.tsx]: if `scriptName` is undefined should we not try to navigate? A way you can test this is to go to the `/levels` url for a level (accessible via the extra links box)

**Review by molly-moen — COMMENTED** [review]
re: adding to all the things, I don't think you need to, but it's generally safe to do via a file generated locally. The main thing to watch out for is if you haven't seeded in a while and you accidentally delete a level, then you'll want to undo the changes and re-seed so you don't delete anything. But it'll often cause harmless numbering changes, see [an example](https://github.com/code-dot-org/code-dot-org/pull/59683/files#diff-10931e7fe766c3a0f8df609d130ee9daa32965a6208bfa1daa6e94e788f78b5a).

File: apps/src/codebridge/InfoPanel/ValidatedInstructions.tsx (hunk 1 above)
Comment (cnbrenci) [reply apps/src/codebridge/InfoPanel/ValidatedInstructions.tsx]: Thanks for the tips! Tested through the /levels url and validated that there's a no-op now in that case.

**Review by molly-moen — APPROVED** [review]
🎉

**Pull request merged**